src/bun.js/bindings/sqlite/JSSQLStatement.cpp

```diff
@@ -273,6 +285,37 @@ static JSValue createSQLiteError(JSC::JSGlobalObject* globalObject, sqlite3* db)
     return object;
 }
 
+class SQLiteBindingsMap {
+public:
+    SQLiteBindingsMap() = default;
+    SQLiteBindingsMap(JSC::VM& vm, sqlite3_stmt* stmt, bool trimLeadingPrefix = false, size_t count = 0)
+    {
+        bindingNames.reserveInitialCapacity(count);
+        hasLoaded = true;
+        size_t prefixOffset = trimLeadingPrefix ? 1 : 0;
+
+        for (size_t i = 0; i < count; i++) {
+            const unsigned char* name = reinterpret_cast<const unsigned char*>(sqlite3_bind_parameter_name(stmt, i + 1));
+            if (name == nullptr) {
+                break;
```

Comment:
should this be `continue` instead of `break`. docs say `NULL` is returned if out of range or if it's nameless